scripts/prettier/index.js

```diff
@@ -52,11 +48,17 @@ const config = {
 
 function exec(command, args) {
   console.log('> ' + [command].concat(args).join(' '));
-  var options = {};
-  return execFileSync(command, args, options).toString();
+  var options = {
+    cwd: process.cwd(),
+    env: process.env,
+    stdio: 'pipe',
+    encoding: 'utf-8',
+  };
+  const childProcess = spawnSync(command, args, options);
+  return childProcess.stdout.trim();
```

Comment:
Can we leave the .trim() in the callers as before? You might not want to trim it always.